### PR TITLE
[ZEN-4601] Move cmr.gentemplates.swagger.codegen.test project to GenFlow

### PR DESCRIPTION
**Pull request by ghillairet**
This PR migrates the swagger codegen tests from API studio to GenFlow. The tests are failing now due to issues in some swagger generator. Updating to newer version of swagger-codegen would resolve some of them.

### CODE REVIEW

**Comment by tedepstein** [comment]
We need to move our SCG v2 to the newer v2.5.2 release. Do it as a separate PR first.

**Comment by ghillairet** [comment]
@nbhusare This needs only a code review, no testing on API Studio as it only adds a test.

File: modules/gentemplates/swagger-codegen/src/test/java/com/reprezen/genflow/swagger/codegen/SwaggerCodegenModulesTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright © 2013, 2016 Modelsolv, Inc.
```
Comment (nbhusare) [inline modules/gentemplates/swagger-codegen/src/test/java/com/reprezen/genflow/swagger/codegen/SwaggerCodegenModulesTest.java]: This is a new file. Should we change this to 2019?

Comment (tedepstein) [reply modules/gentemplates/swagger-codegen/src/test/java/com/reprezen/genflow/swagger/codegen/SwaggerCodegenModulesTest.java]: @nbhusare , Yes, we should update our copyright notices to say 2013-2019:

>  Copyright © 2013, 2016 Modelsolv, Inc.

I'll open a Jira for this so we can make this change across all of our open source projects.

**Pull request merged**